content/docs/next-release-v2/frontend-ui.md

```diff
@@ -146,6 +151,17 @@ For traces, the supported template fields are: `duration`, `endTime`, `startTime
 
 Further, the trace template fields are available for substitution in process/logs/tags type when the trace template fields are prefixed with `trace.`. For example: `trace.traceID`, `trace.startTime`.
 
+#### Formatting
+
+In addition to interpolating fields into links, formatting functions can be used. The syntax is `#{field | function}` (eg `#{endTime | epoch_micros_to_date_iso}'`). The available formatting functions and a description of their behavior are:
+| Function name | Arguments | Description | Example |
```

Comment:
Table results it very bad rendering where it bleeds into the RHS sidebar.

Let's use normal markup. It also makes function names into anchors.
```
##### pad_start
Pad the start of a string with a given character until the resulting string reaches the given length. Behavior is implemented by and thus matches...

Arguments:
  * `targetLength` (number) - ...
  * `padCharacter` (string) - ...
```
##### pad_start
Pad the start of a string with a given character until the resulting string reaches the given length. Behavior is implemented by and thus matches...

Arguments:
  * `targetLength` (number) - ...
  * `padCharacter` (string) - ...